search/utils.py

```diff
@@ -0,0 +1,23 @@
+import re
+from typing import Union
+
+import flask
+
+import requests
+
+
+def strip_slashes(path: str) -> str:
+    """removes forward slashes from the beginning and the end of a string"""
+    return re.sub(r'^/|/$', '', path)
+
+
+def get_content(path: str, id: Union[int, str] = None) -> bytes:
```

Comment:
I think the spec defines ids as strings (they may look like integers but it's a string containing numerical characters)

Reply:
ah ok. That makes this simpler then. I'll update.